packages/uix-host/src/port.ts

```diff
@@ -435,10 +435,7 @@ export class Port<GuestApi = unknown>
       try {
         methodCallee = this.getHostMethodCallee(address, privateMethods);
       } catch (e) {
-        this.logger.warn(
-          `Private method ${formatHostMethodAddress(address)} not found!`,
-          address
-        );
+        // ignore error, privateMethod gets called and works
       }
     }
     if (!methodCallee) {
```

Comment:
This sounds good? Error triggered only if the methodCalee is really not found using both methods `this.getHostMethodCallee(address, privateMethods)` and `this.getHostMethodCallee(address, this.hostApis)`. It will be an `error` because its not possible to proceed if methodCalee is unavailable.

```
let methodCallee;
    if (privateMethods) {
      try {
        methodCallee = this.getHostMethodCallee(address, privateMethods);
      } catch (e) {
        // continue and try other approach to get methodCallee
      }
    }
    if (!methodCallee) {
      methodCallee = this.getHostMethodCallee(address, this.hostApis);
    }

    if(methodCallee){
      const method = methodCallee[name] as (...args: unknown[]) => T;
      ...
      return method.apply(methodCallee, [
        ...
      ]) as T;
    } else {
      this.logger.error(
        `Private method ${formatHostMethodAddress(address)} not found!`,
        address
      );
    }
 ```

Reply:
yes, looks good to me 👍

Reply:
thx will test this change and merge

Reply:
I think `methodCallee = this.getHostMethodCallee(address, this.hostApis);` will trigger an exception if the method is missing so there is no need to log the error afterwards

Reply:
@kokoc how about handle this exception instead using catch, so that other functionality may remain unaffected incase there is such an exception? keeping it unhandled might affect something else.